ovis_moveit_config/config/ros_controllers.yaml

```diff
@@ -0,0 +1,25 @@
+# Simulation settings for using moveit_sim_controllers
+moveit_sim_hw_interface:
+  joint_model_group: manipulator
+  joint_model_group_pose: right_angle
+# Settings for ros_control_boilerplate control loop
+generic_hw_control_loop:
+  loop_hz: 300
+  cycle_time_error_threshold: 0.01
+# Settings for ros_control hardware interface
+hardware_interface:
+  joints:
+    - arm_joint_1
+    - arm_joint_2
+    - arm_joint_3
+    - arm_joint_4
+    - arm_joint_5
+    - arm_joint_6
+  sim_control_mode: 1  # 0: position, 1: velocity
+# Publish all joint states
+# Creates the /joint_states topic necessary in ROS
+joint_state_controller:
+  type: joint_state_controller/JointStateController
+  publish_rate: 50
+controller_list:
+  []
```

Comment:
Is the controller_list necessary if empty?

Reply:
We do not use this file, it's generated by the moveit wizard. Ovis controller configurations are stored in the "ovis_bringup/config/ovis_controllers.yaml" file.